### PR TITLE
add logging for invalid TASK_INTERVAL

### PR DESCRIPTION
**Pull request by vtolstov**
Signed-off-by: Vasiliy Tolstov <v.tolstov@selfip.ru>

### CODE REVIEW

File: modules/setting/setting.go [hunk 1]
```diff
@@ -548,6 +548,7 @@ func newWebhookService() {
 	sec := Cfg.Section("webhook")
 	td, err := time.ParseDuration(sec.Key("TASK_INTERVAL").String())
 	if err != nil {
+		log.Warn("invalid TASK_INTERVAL value %s %s", sec.Key("TASK_INTERVAL").String(), err.Error())
```
Comment (tboerger) [inline modules/setting/setting.go]: ```Invalid```, uppercase at the beginning of the logging line :)

**Comment by tboerger** [comment]
Small NITPICK, but otherwise :+1:

**Comment by vtolstov** [comment]
@tboerger as i see many opensource projects does not use uppercase for logging.

**Comment by tboerger** [comment]
I think that is a weak argument just because others are doing it that way...

**Pull request merged**